src/calendarevent.cpp

```diff
@@ -248,17 +241,17 @@ void CalendarStoredEvent::notebookColorChanged(QString notebookUid)
 
 void CalendarStoredEvent::eventUidChanged(QString oldUid, QString newUid)
 {
-    if (mData->uniqueId == oldUid) {
-        mData->uniqueId = newUid;
-        emit uniqueIdChanged();
+    if (mData->instanceId == oldUid) {
```

Comment:
It's legacy code. As you notice, it's currently dead code because the manager is never emitting the signal.

I'm not touching it (besides member naming change) for the moment.